internal/onepassword/cli/op.go

```diff
@@ -208,6 +208,20 @@ func (op *OP) delete(ctx context.Context, item *onepassword.Item, vaultUuid stri
 	return nil, op.execJson(ctx, nil, nil, p("item"), p("delete"), p(item.ID), f("vault", vaultUuid))
 }
 
+func (op *OP) GetFileContent(ctx context.Context, file *onepassword.File, itemUuid, vaultUuid string) ([]byte, error) {
```

Comment:
`[nitpick]`

This will make the CLI client part of the Terraform provider slightly less performant, since `GetItem` (as in `op item get`) already gets the content of the files the item has. So now what will happen is to fetch the item twice instead of once.

Reply:
I actually tried this, and when running tests everything looked good, but when I actually tried using the provider (with op cli) i got the content not yet loaded error, so it did not work